src/main/java/com/bootcamp/demo/model/util/Battery.java

```diff
@@ -0,0 +1,48 @@
+package com.bootcamp.demo.model.util;
+
+import java.util.Objects;
+
+public final class Battery {
+    private Double level;
+    private BatteryStatus status;
```

Comment:
Include the validation in this PR?

Reply:
I was not thinking from a validation perspective, but rather from a modelling one. Why not have the level encapsulated in `BatteryStatus`?

Reply:
The BatteryStatus is an enum with a fixed string category name("LOW", "MEDIUM", "HIGH"). If I add the level (Double) attribute, then, for ex. I want to change the category name because the level is not compatible with the present one, then  I can't do that because its fixed. 

Or, I don't understand how do you want to do that. Can you explain it more?

> Enum constants are implicitly static and final and you can not change their value once created. Enum in Java provides type-safety and can be used inside switch statements like int variables. 

I tried this: 
` public enum BatteryStatus {
    LOW("LOW", 0.0),
    MEDIUM("MEDIUM", 30.0),
    HIG("HIGH", 60.0);

    private final String value;
    private Double level;

    BatteryStatus(String value, Double level) {
        this.value = value;
        this.level = level;
    }

    public Double getLevel() {
        return level;
    }

    public String getValue() { return value; }
}
`